renovate.json

```diff
@@ -1,53 +1,30 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "platform": "github",
-  "labels": ["renovate"],
   "extends": [
-    "config:base",
-    ":rebaseStalePrs"
+    "github>gsoft-inc/renovate-config"
   ],
-  "enabledManagers": [
-    "github-actions",
-    "nuget"
-  ],
-  "stabilityDays": 3,
-  "prHourlyLimit": 0,
-  "prConcurrentLimit": 0,
-  "branchConcurrentLimit": 0,
-  "dependencyDashboard": false,
-  "gitAuthor": "Renovate Bot <bot@renovateapp.com>",
   "packageRules": [
     {
-      "matchManagers": ["nuget"],
-      "excludePackagePatterns": ["^Microsoft\\.Extensions\\.", "^System\\.", "^dotnet-sdk$"],
-      "groupName": "NuGet dependencies"
-    },
-    {
-      "matchManagers": ["nuget"],
-      "matchPackagePatterns": ["^Microsoft\\.Extensions\\.", "^System\\."],
+      "matchManagers": [
+        "nuget"
+      ],
+      "matchPackagePatterns": [
+        "^Microsoft\\.Extensions\\.",
+        "^System\\."
+      ],
       "groupName": "Ignored NuGet dependencies",
       "description": "These packages are usually set to a user-defined minimal supported version such as 6.0.0 for .NET 6, and they are overriden by consuming applications",
       "enabled": false
     },
     {
-      "matchManagers": ["nuget"],
-      "excludePackageNames": ["xunit"],
+      "matchManagers": [
+        "nuget"
+      ],
+      "excludePackageNames": [
+        "xunit"
```

Comment:
Remove this rule

Reply:
It was causing issue to debug Linting rules, has it changed?

Reply:
I think this is because of Roslyn analyzers. In this case, update the code to use the right packages. We did it in multiple repositories (e.g. https://github.com/gsoft-inc/wl-extensions-mongo/commit/9ba066e4b3b5ec0847b5d82cd702ead4d8a4891f#diff-9ec05a9128808e24edbda1b22dc753726d4f8ca87c875e8771e796e43d17a8b4R14)